### ISSUE TITLE
Setup live credentials

### ISSUE THREAD

**Issue by pablofullana** [poanetwork/poa-poba#19]

**Comment by matiasgaratortiz** [comment]
Hi @igorbarinov, I had sent an email to Plaid support asking about credentials for `dev`. But, they asked me about that:

```Hi there,Thank you for your interest in Plaid!We are noticing international traffic associated with your dashboard account. Can you please confirm the source of that traffic? Additionally, can you confirm that any data associated with the use of Plaid products will only be transmitted and/or stored within the United States?Thank you and please let us know if you have any question
```

**Comment by igorbarinov** [comment]
> We are noticing international traffic associated with your dashboard account. Can you please confirm the source of that traffic.

you are based in Argentina

> Additionally, can you confirm that any data associated with the use of Plaid products will only be transmitted and/or stored within the United States?

Only U.S. based users will access their bank account in the U.S.

You can forward them to my email

**Comment by fvictorio** [comment]
The "stored" part is the problematic one, I think, if we are going to store the account numbers in the blockchain.

And if the "international traffic" is a problem, then we are going to have a hard time testing it, but I guess just clarifying that will be enough.

**Comment by matiasgaratortiz** [comment]
Yes, I think the same. If we clarify and explain that we are going to be testing from Argentina, the international traffic could be solved and would not be a problem. The point is then the "data will only be transmitted and/or stored within the United States"
